auth-web/src/views/auth/create-account/AccountChangeView.vue

```diff
@@ -41,14 +43,14 @@ export default class AccountChangeView extends Vue {
         componentProps: { 'isAccountChange': true, 'cancelUrl': ConfigHelper.accountSettingsRoute() }
       },
       {
```

Comment:
These stepper changes should be needed in the create account steps as well right?